src/print.rs

```diff
@@ -289,7 +289,7 @@ impl BacktraceFrameFmt<'_, '_, '_> {
             write!(self.fmt.fmt, ":{colno}")?;
         }
 
-        write!(self.fmt.fmt, "\n")?;
+        writeln!(self.fmt.fmt)?;
```

Comment:
I ran `cargo test` with the newline replaced with `""` here and there do not appear to be any tests covering this code.

Reply:
Thanks.